glue/core/message.py

```diff
@@ -28,15 +28,23 @@ class Message(object):
     The message class family is hierarchical, and a client subscribing
     to a message class implicitly subscribes to all of its subclasses.
 
-    :attr sender: The object which sent the message
-    :attr tag: An optional string describing the message
+    Attributes
+    ----------
+    sender :
```

Comment:
When there is no specific type (as is the case here), just omit the ``:``.

Reply:
Is there a way to find out the type for a parameter that I am unaware of? Any `type` equivalent which can be used?